dropwizard-jersey/src/main/java/io/dropwizard/jersey/errors/ErrorMessage.java

```diff
@@ -1,5 +1,8 @@
 package io.dropwizard.jersey.errors;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
```

Comment:
Can we also add `@JsonInclude(JsonInclude.Include.NON_NULL)` in here to avoid having null fields show up in the error responses if they aren't being set? (I ran into this with the `details` field)